### PR TITLE
Added support for onStartTrackingTouch, onStopTrackingTouch.

### PR DESCRIPTION
**Pull request by natami**
Let me know, if it's of any use ;)

### CODE REVIEW

File: lib/src/main/java/io/apptik/widget/MultiSlider.java [hunk 1]
```diff
@@ -1035,8 +1037,10 @@ public boolean onTouchEvent(MotionEvent event) {
                         invalidate(currThumb.getThumb().getBounds()); // This may be within the padding region
                     }
 
-                    setThumbValue(currThumb, getValue(event, currThumb), true);
+                    int value = getValue(event, currThumb);
+                    setThumbValue(currThumb, value, true);
                     attemptClaimDrag();
+                    mOnThumbValueChangeListener.onStartTrackingTouch(this, currThumb, value);
```
Comment (djodjoni) [inline lib/src/main/java/io/apptik/widget/MultiSlider.java]: we have a method onStartTrackingTouch where thumbs are set as being dragged and their state is changed.
This is the place where the callback should be called. 
in fact 'attemptClaimDrag()' should also be called there

File: lib/src/main/java/io/apptik/widget/MultiSlider.java [hunk 2]
```diff
@@ -1111,8 +1115,10 @@ public boolean onTouchEvent(MotionEvent event) {
             //we normally have one single pointer here and its gone now
             case MotionEvent.ACTION_UP:
                 if (currThumb!=null) {
-                    setThumbValue(currThumb, getValue(event, currThumb), true);
+                    int value = getValue(event, currThumb);
+                    setThumbValue(currThumb, value, true);
                     onStopTrackingTouch();
+                    mOnThumbValueChangeListener.onStopTrackingTouch(this, currThumb, value);
```
Comment (djodjoni) [inline lib/src/main/java/io/apptik/widget/MultiSlider.java]: same as before, it is better to call in this inside onStopTrackingTouch()

Comment (natami) [inline lib/src/main/java/io/apptik/widget/MultiSlider.java]: I will se if i can find some time during the week :)
Den 25/10/2015 10.20 PM skrev "Kalin Maldzhanski" <notifications@github.com

**Comment by natami** [comment]
Let me know if it's ok now :)

**Comment by djodjoni** [comment]
Thx!

**Pull request merged**